cedar-policy-core/src/entities/json/err.rs

```diff
@@ -120,6 +120,11 @@ pub enum JsonDeserializationError {
     #[error(transparent)]
     #[diagnostic(transparent)]
     TypeMismatch(TypeMismatch),
+    /// When trying to deserialize an AST with an error in it - this should fail
+    #[cfg(feature = "tolerant-ast")]
+    #[error("Unable to deserialize an AST Error node")]
+    #[diagnostic(help("AST error node indicates that the expression has failed to parse"))]
+    ASTErrorNode,
```

Comment:
Was this variant intended to carry an `ErrorNode`?

Reply:
Yup, will fix

Reply:
Actually, on second thought the `ErrorNode` doesn't really add anything because the context is always just going to end up being unknown anyway. I'll remove it entirely